target/default+test+test/classes/META-INF/maven/tic-tac-toe/tic-tac-toe/pom.properties

```diff
@@ -0,0 +1,6 @@
+#Leiningen
```

Comment:
You shouldn't include any target files. can you remove them from git and gitignore them?

Reply:
Yeah I just did the same above